onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -48,7 +48,8 @@ static const OpVersionsAndSelector::OpVersionsMap GetMiscOpVersionsMap() {
 // These produce int64 indices output, which can't be quantized, so there's no downstream Q node.
 static const OpVersionsAndSelector::OpVersionsMap GetDropDQOpVersionsMap() {
   return {{"ArgMax", {}},
-          {"ArgMin", {}}};
+          {"ArgMin", {}},
+          {"NonZero", {}}};
```

Comment:
>{"NonZero", {}}} [](http://example.com/codeflow?start=10&length=16)

This would apply to all NonZero then, not just QNN?

Reply:
Is it safe to drop the DQ? Wouldn't the zero point and scale affect which values are zero?